taf/tools/repo/__init__.py

```diff
@@ -181,7 +181,7 @@ def initialize(path, library_dir, namespace, targets_rel_dir, custom, use_mirror
 
     @repo.command()
     @click.argument("url")
-    @click.option("--clients-auth-path", default=None, help="Directory where authentication repository is located.")
+    @click.argument("clients-auth-path", default=None, required=False)
```

Comment:
The reason why we tend to use options is because it allows us to use the help parameter, which results in nice `--help` output. I can see why it would be nice to lose `--`, but then we need to update the docstring and explain clients-auth-path's purpose

Reply:
Thought it makes sense to transition taf back to old implementation that is similar to how `git clone` works. `git clone` has a URL and a second (optional) argument. The second argument made it possible to clone to current directory. So for example `git clone https://github.com/openlawlibrary/taf .` clones to current directory without specifying any flags. What do you think?

Reply:
Yeah, that's easier to use, but we should make sure that docstring is updated following removal of help.